### PR TITLE
next urql client

### PR DESCRIPTION
**Pull request by TonySchroeder**
close https://github.com/progwise/george-ai/issues/46

### CODE REVIEW

File: apps/nextjs-web/app/layout.tsx [hunk 1]
```diff
@@ -1,22 +1,38 @@
...
   children: React.ReactNode
 }) {
   return (
-    <html lang="en">
-      <body className={inter.className}>{children}</body>
-    </html>
+    <UrqlProvider client={client} ssr={ssr}>
+      <html lang="en">
+        <head>
+          <title>{metadata.title}</title>
```
Comment (PascalHelbig) [inline apps/nextjs-web/app/layout.tsx:31]: There is an official way for setting metadata: https://nextjs.org/docs/app/building-your-application/optimizing/metadata

File: apps/nextjs-web/app/page-list.tsx [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
 import { InfoCard } from './components/info-card/info-card'
-import fetchData from './fetch-data'
+import useFetchData from './fetch-data'
+import Loading from './loading'
+import { Page } from './page'
+
+export const PageList = () => {
+  const result = useFetchData()
+
+  if (result.fetching) return <Loading />
```
Comment (PascalHelbig) [inline apps/nextjs-web/app/page-list.tsx:11]: Why is this not a server component, like this?
image
https://formidable.com/open-source/urql/docs/advanced/server-side-rendering/#nextjs

File: apps/nextjs-web/app/fetch-data.tsx [hunk 3]
```diff
@@ -1,39 +1,29 @@
...
+        feedback
+        generatedKeywords
+        generatedSummary
+        largeLanguageModel
+      }
+    }
   }
+`
+
+const useFetchData = () => {
```
Comment (PascalHelbig) [inline apps/nextjs-web/app/fetch-data.tsx:24]: The name for this hook is not meaningful at all, and also in a file with a complete different name.
Besides, not much really happens in the hook. What do you think about moving the hook into `page-list.tsx`?

**Review by PascalHelbig — APPROVED** [review]
Looks very good on the whole. I have a few questions of detail.

File: apps/nextjs-web/app/layout.tsx (hunk 1 above)
Comment (TonySchroeder) [reply apps/nextjs-web/app/layout.tsx:31]: I have already tried it.
Screenshot 2023-08-16 at 09 20 51

Comment (moncapitaine) [reply apps/nextjs-web/app/layout.tsx:31]: lets stick to client-side for now.

File: apps/nextjs-web/app/page-list.tsx (hunk 2 above)
Comment (moncapitaine) [reply apps/nextjs-web/app/page-list.tsx]: what we would need here is a server-side render the top 10 and then we want to filter the list on every keystroke as it was on http://services.progwise.net:8080/ - is this the reason why it is client-side for now?

**Pull request merged**